_opensearch/install/compatibility.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Compatibility
+parent: Install OpenSearch
+nav_order: 2
+---
+
+# Operating system and JVM compatibility
+
+- We recommend installing OpenSearch on RHEL- or Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, and Ubuntu (LTS). OpenSearch should work on many Linux distributions, but we only test a handful.
+- The OpenSearch tarball ships with a compatible version of Java in the `jdk` directory. To find its version, run `./bin/opensearch --version`.
```

Comment:
We should say which version it is. Currently I believe it's 15.

Reply:
Sure, added.